pipenv/project.py

```diff
@@ -199,6 +199,10 @@ def sources(self):
         else:
             return [{u'url': u'https://pypi.python.org/simple', u'verify_ssl': True}]
 
+    @property
+    def source_urls(self):
```

Comment:
Could you clarify the duplication you're referring to? We're now iterating the sources content twice and creating a second data structure to represent the same data.

If there isn't a particular benefit of the list, it would be better to simply use the `sources` property.